linker/shared/src/main/scala/org/scalajs/linker/analyzer/Analyzer.scala

```diff
@@ -495,7 +535,7 @@ private class AnalyzerRun(config: CommonPhaseConfig, initial: Boolean,
       unvalidatedInterfaces: List[ClassInfo],
       val syntheticKind: Option[SyntheticClassKind],
       val nonExistent: Boolean)
-      extends Analysis.ClassInfo with ClassLoadingState with LoadingResult with ModuleUnit {
+      extends Analysis.ClassInfo with LoadingResult with InfoLoadedState with ModuleUnit {
```

Comment:
This seems redundant. IIUC `LoadingResult extends InfoLoadedState`, so:
```suggestion
      extends Analysis.ClassInfo with LoadingResult with ModuleUnit {
```
?